automatminer/automl/base.py

```diff
@@ -146,9 +153,11 @@ def predict(self, df: pd.DataFrame, target: str) -> pd.DataFrame:
                 "".format(not_in_df, not_in_model)
             )
         else:
+            if target_output_col is not None:
```

Comment:
can we just have the target output col argument be the "{target} predicted" string? Having it be None makes it a bit harder to see what is going on

Reply:
oh nvm, I see it is set in the base class constructor